script/cibuild

```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
 set -e
 
-bundle exec jekyll build
+if [ $1 = "production" ]; then
```

Comment:
```suggestion
if [ "$1" = "production" ]; then
```

Reply:
@jasoncpatton Yes, that is a good call. Thank you for the suggestion!